requirements.txt

```diff
@@ -1,6 +1,6 @@
 bert-score>=0.3.5
 editdistance
-flair==0.6.1.post1
+flair
```

Comment:
Adding a note here just in case PyTorch versions break this requirement in the future.